components/Board.js

```diff
@@ -0,0 +1,9 @@
+import { Chessboard } from "react-chessboard";
+
+export default function Board() {
+  return (
+    <div>
+      <Chessboard id="BasicBoard" />
+    </div>
```

Comment:
do you need a div here? If you don't, you should remove it.

and what purpose serves the id? If it serves no purpose (yet), you should remove it.

Reply:
True, atm I don´t need the div. I tried some stylings that didn't work out, thats why there are left there. I´ll remove them for the time being an reintroduce them  if necessary